### ISSUE TITLE
[BUG] Dual Monitor freezes system at boot

### ISSUE THREAD

**Issue by madmurl0c** [randyzhong/OS-X-NUC8i7BEH-Clover#1]
@randyzhong first of all thanks for this great configuration, even the internal Microphones are working, really impressive 👍

There's still one problem I wasn't able to solve. I have two screens attached (1x HDMI & 1x USB-C) and they both work fine once the system is booted up but in order to boot the system I need to disconnect one of the monitors, otherwise it will freeze at the apple logo with the progress bar. Do you know a fix for this?

Edit: First I thought it was because of the USB-C to DisplayPort adapter but this is not the case

**Comment by randyzhong** [comment]
@madmurl0c Thank for your testing and feedback. So you if you disconnect the HDMI monitor and only get USB-C over DP monitor connecting, it can be booted up successfully? 

Currently I don't have a USB-C to DP/HDMI adapter, let me get one next week and see if I can find anything.

**Comment by madmurl0c** [comment]
@randyzhong I don't have my NUC anymore (switched to mac mini) but it worked with only USB-C or HDMI attached. The error only occured when both screens were attached while booting.

**Comment by randyzhong** [comment]
@madmurl0c I purchased a USB Type-C to DP adapter and encountered the same problem as you. After fighting with the Framebuffer patches for almost a day, sometimes it can boot successfully with dual monitors, but next restart fails.  I gave up with patches on platform-id 0x3ea50004,  however, if enabled **FixIntelGfx** and **AddHDMI**,  found it's working but with platform-id 0x3e9b0007, eventually found this [back to 0x3ea50000; place alternate ig-platform-id into SSDT_CFLALT](https://github.com/RehabMan/Intel-NUC-DSDT-Patch/commit/e3cd2b51c080abbfca1db2a2ae09e0b16e6e49f5)

Thanks to @RehabMan.

```
// SSDT_CFLALT.dsl
//
// Alternate configuration for systems where 0x3e9b0007 works better than 0x3ea50000.
//

DefinitionBlock("", "SSDT", 2, "hack", "_CFLALT", 0)
{
    Name(RMGO, Package()
    {
        // Coffee Lake/UHD655
        0x3ea5, 0, Package()
        {
            "AAPL,ig-platform-id", Buffer() { 0x07, 0x00, 0x9b, 0x3e },
            "hda-gfx", Buffer() { "onboard-1" },
            "model", Buffer() { "Intel Iris Plus Graphics 655" },
        },
    })
}
//EOF
```

So compile this to SSDT_CFLALT.aml and copy to EFI/CLOVER/ACPI/patched/ folder fixes this issue. I will update my repository later. Thank you for the feedback.
